internal/debug/flags.go

```diff
@@ -60,26 +60,26 @@ var (
 		Usage: "Enable the pprof HTTP server",
 	}
 	pprofPortFlag = cli.IntFlag{
-		Name:  "pprofport",
+		Name:  "pprof.port",
 		Usage: "pprof HTTP server listening port",
 		Value: 6060,
 	}
 	pprofAddrFlag = cli.StringFlag{
-		Name:  "pprofaddr",
+		Name:  "pprof.addr",
```

Comment:
The flag changes need to be recorded for when we make the next release so we can warn operators that we will be following upstream in deprecating certain flags.